schema_cache.rst

```diff
@@ -106,26 +111,52 @@ To refresh the cache without restarting the PostgREST server, send the server pr
      # or in docker-compose
      docker-compose kill -s SIGUSR1 <service>
 
-The above is the manual way to do it. To automate cache reloads, use a database trigger like this:
+.. _schema_reloading_notify:
+
+NOTIFY
+~~~~~~
+
+Send a `database notification <https://www.postgresql.org/docs/current/sql-notify.html>`_ from any PostgreSQL client by executing the ``NOTIFY`` command as follows:
 
 .. code-block:: postgresql
 
-  CREATE OR REPLACE FUNCTION public.notify_ddl_postgrest()
-    RETURNS event_trigger
-   LANGUAGE plpgsql
+  NOTIFY pgrst
+
+  -- Works the same way as:
+
+  NOTIFY pgrst, 'reload schema'
```

Comment:
I'd prefer to be explicit here, the empty body NOTIFY might change in the future.
```suggestion
  NOTIFY pgrst, 'reload schema'
```